lv2/sfizz.ttl.in

```diff
@@ -14,6 +14,10 @@
 @prefix units:   <http://lv2plug.in/ns/extensions/units#> .
 @prefix urid:    <http://lv2plug.in/ns/ext/urid#> .
 @prefix work:    <http://lv2plug.in/ns/ext/worker#> .
+@prefix midnam:  <http://ardour.org/lv2/midnam#> .
+
+midnam:interface a lv2:ExtensionData .
+midnam:update a lv2:Feature .
```

Comment:
More of a lack of knowledge on my part than a comment, but do we need to declare these? I didn't add these for e.g. `state:interface` or `work:interface`.

Reply:
These are defined in TTL files part of the LV2 distribution.

Ardour does not set up some global files that define these objects, it's the reason for doing it here. It's from gmsynth.lv2.